backend/application.py

```diff
@@ -40,6 +42,83 @@ def build_dataset_structure(dataset_version, user=None, dataset=None):
     return r
 
 
+class GetSchema(handlers.UnsafeHandler):
+    """
+    Returns the schema.org, and bioschemas.org, annotation for a given
+    url.
+
+    This function behaves quite differently from the rest of the application as
+    the structured data testing tool had trouble catching the schema inject
+    when it went through AngularJS. The solution for now has been to make this
+    very general function that "re-parses" the current uri to figure out what
+    information to return.
+    """
+    def get(self):
+
+        dataset = None
+        version = None
+        match = re.match(".*\?url=(.*)",  unquote(self.request.uri) )
```

Comment:
Why are you using `re.match` on the URL instead of the tornado method `get_argument`? Like this

```
url = self.get_argument('url')
```

http://www.tornadoweb.org/en/stable/web.html#tornado.web.RequestHandler.get_argument


And codacy was complaining that you weren't using `r""` (the `r`) in your regexes.